Gemfile

```
@@ -4,6 +4,7 @@ ruby "2.6.3"
 gem "bcrypt", "3.1.15"
 gem "bootsnap", ">= 1.4.2", require: false
 gem "config"
+gem "faker", git: "https://github.com/faker-ruby/faker.git", branch: "master"
```

```suggestion
gem "faker"

```
gem này dùng làm gì?

---

để fake data như tên tuổi email địa chỉ ....